dotCMS/src/curl-test/Workflow Resource Tests.json

```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "cd87df22-9b12-4ee0-b05f-290155476363",
+		"_postman_id": "fd766073-2359-456c-99ae-2563435c67e4",
```

Comment:
when you modified this file, did you make sure you were working with the latest version? it seems that some tests are being removed with your commit

Reply:
some curl I have done, didn't make sense at the end